src/components/radio-button/radio-button-group/radio-button-group.ts

```diff
@@ -304,15 +304,14 @@ export class SbbRadioButtonGroup extends LitElement {
       return;
     }
 
-    let current: number;
```

Comment:
I think we have still an open issue: If you focus into the group, the currently selected option should be focused and not the first (but again, only in case of all panels do not have content). Afterwards I think we are close to the behavior of a radio button :)

Reply:
See commit 15edb6f